12runningSS.tex

```diff
@@ -140,12 +142,22 @@ \subsubsection{Running Without Estimation}
 
 where -1 is the maximum phase for the model to run through (e.g., can be other values if a user would like to only run through a specific parameter phase). This approach will create all the new files with the starter.ss\_new reflecting the original maxphase value in the starter.ss file. This approach is available in v.3.30.16 and later.
 
+\hypertarget{hess-step}{}
+\subsubsection{Using -hess\textunderscore step to do additional Newton steps using the inverse Hessian}
+
+The optimizer in ADMB is designed to run until the maximum absolute gradient (mac) is small enough (e.g., 1e-05), after which it quits and does the uncertainty calculations. But if run for longer it cannot appreciably decrease this mac. In many cases it is interesting or advisable to get closer to the mode to confirm convergence of the model. 
```

Comment:
"mac" should be "mag" here and below, I think. And maybe better as "MAG" (which would be consistent with "MLE".